### ISSUE TITLE
MBDebouncedTextField fails to render inside EditForm component

### ISSUE THREAD

**Issue by skb50bd** [Material-Blazor/Material.Blazor#340]
When `MBDebouncedTextField` is used inside an `EditForm` Component, it fails to render and throws an exception.

**Comment by simonziegler** [comment]
Thanks for this. I'll test it out. Can you let me know if you are using a Validate component after it (or using our built in variety)?

However debounced theft fields aren't intended for use inside edit forms and are supposed to disable edit form functionality. Imagine if you were in a form and typed part of a word and then paused for 300ms - you'd be at risk of an immediate validation error.

We're improving this mechanism in the upcoming version 2 and will log a warning if you do put a debounced text field in a form.

In short, we intend these to be used for things like search functionality.

**Comment by simonziegler** [comment]
@MarkStega and @simonziegler to discuss pending closing this issue.

**Comment by simonziegler** [comment]
We have decided to leave this as it is, with `MBDebouncedTextField` not operating within an `EditForm`, This is documented at https://material-blazor.com/docs/Material.Blazor/Components/DebouncedTextField/MBDebouncedTextField.html.
